### PR TITLE
output once mode

### PR DESCRIPTION
**Pull request by wheresrhys**

### CODE REVIEW

**Comment by wheresrhys** [comment]
@kaelig

File: main.scss [hunk 1]
```diff
@@ -53,8 +53,14 @@
 }
 
 @if $o-grid-is-silent == false {
-	@include oGridSurfaceCurrentLayout;
-	@include oGridGenerate;
+
+	// only output if either not outputted already or multiple outputs are allowed
+	@if $o-grid-output-once == false || $_o-grid-outputted == false {
```
Comment (kaelig) [inline main.scss]: In Sass, the `||` operator is written `or`

**Comment by kaelig** [comment]
I understand the frustration, but this looks like an edge case that only Next has encountered so far, so should this be solved that at a product level instead?

For example the product could include this directly:

```scss
@include oGridSurfaceCurrentLayout;
@include oGridGenerate;
```

**Comment by kaelig** [comment]
Closing as we've found a compromise that will suit Next for the moment.

**Pull request closed without merging**